cloudigrade/util/aws/sts.py

```diff
@@ -19,6 +19,17 @@
                 'ec2:DescribeSnapshots'
             ],
             'Resource': '*'
+        },
+        {
+            'Effect': 'Allow',
+            'Action': [
+                'cloudtrail:StartLogging',
```

Comment:
Is there a reason these are separate from the above permissions? Having it as a separate section bypasses the `aws.helper.verify_account_access()` function. We should combine the two, and also write out action test cases in `_verify_policy_action()`.